ush/remap.sh

```diff
@@ -0,0 +1,376 @@
+#     <scalar variable> <CDO remapping type> <input variable netCDF file> 0
+#     <vector variables> <CDO remapping type> <input variable netCDF file> 1
+#        <grid rotation variable(s)>
+#
+#       An example using the format described above is as follows.
+#
+#     aice_h remapbil /path/to/ice/model/file 0
+#     uo,vo remapbil /path/to/ocean/model/file 1 cos_rot,sin_rot
+#
+#       The example above will perform the following tasks using this
+#       script.
+#
+#       * Remap the ice model `aice_h` scalar variable, **without**
+#         rotation, using the CDO `remapbil` type;
+#
+#       * Rotate the ocean model current velocity vectors `uo` and
+#         `vo` using the `cos_rot` and `sin_rot` values within the
+#         ocean model netCDF file and subsequently remap them using
+#         the CDO `remapbil` type.
+#
+#       The tested CDO interpolation types are as follows.
+#
+#       * remapbil: bilinear interpolation;
+#       * remapcon: conservative remapping;
+#       * remapnn: nearest-neighbor interpolation.
+#
+#     dstgrid_config: A netCDF-formatted or CDO-type grid description
+#                       file; and example of such a CDO-type grid
+#                       description file, to remap a source grid
+#                       projected variable to a nominal 1p0 grid
+#                       projection is as follows.
+#
+#     gridtype = lonlat
+#     xsize    = 360
+#     ysize    = 181
+#     xfirst   = -179.0
+#     xinc     = 1.0
+#     yfirst   = -90.0
+#     yinc     = 1.0
+#
+#       The latter format is preferred as it provides both readability
+#       and ease of destination grid configuration.
+#
+#     output_netcdf: A netCDF-formatted file path to contain the
+#                      specified variables remapped to the destination
+#                      grid projection.
+
+#######
+
+# Collect the command line arguments and check the validity.
+variable_file="${1}"
+dstgrid_config="${2}"
+output_path="${3}"
+
+#######
+
+if [[ "$#" -ne 3 ]]; then
+    echo "FATAL ERROR: invalid argument syntax provided to ${BASH_SOURCE[0]}"
+    echo "Usage: ${BASH_SOURCE[0]} <variable_file> <input_path> <output_path>"
+    exit 100
+fi
+
+#######
+
+# _comma_split_string - Split a comma-delimited string into an array.
+#
+# Description:
+#   This function takes a comma-delimited string as input and splits
+#   it into an array. Each element in the resulting array is obtained
+#   by splitting the input string at commas and then removing leading
+#   and trailing spaces.
+#
+# Parameters:
+#   $1 - The comma-delimited string to split.
+#
+# Global Variables:
+#   global_array - An array containing the split elements.
+#
+# Example usage:
+#   _comma_split_string "item1,item2 item3,item4"
+#   for element in "${global_array[@]}"; do
+#       echo "$element"
+#   done
+#
+# This example will split the input string into individual elements
+# and print each element on a separate line.
+function _comma_split_string() {
```

Comment:
This function and the next appear in both PRs. These should be moved to a separate script that gets called to avoid duplication.

Will skip reviewing this copy since I've already reviewed those in the other PR.

Reply:
Can you suggest where beneath `/ush` I am assuming to add this?

Reply:
Just put it directly in `/ush` for now.

Reply:
Done.